cocos/input/input.ts

```diff
@@ -229,6 +241,38 @@ export class Input {
         }
         this._eventTarget.off(eventType, callback, target);
     }
+
+    /**
+     * @en
+     * Get touch object by touch ID.
+     * @zh
+     * 通过 touch ID 获取 touch对象。
+     */
+    public getTouch (touchID: number): Touch | undefined {
```

Comment:
I think it is better to return a readonly object.

Reply:
And it is not a good idea to add Input.getTouch() as Input is used for all types input event, such as touch, keyboard and so on.

Reply:
> And it is not a good idea to add Input.getTouch() as Input is used for all types input event, such as touch, keyboard and so on

I think it's OK to add this query interface, latter we could also add `getKeyDown() getKeyUp()` interface to support querying keyboard key state

Reply:
> I think it is better to return a readonly object.

Readonly is useless for a Touch object, we provide setting method on Touch such as `setPoint()` `setPrevPoint()`, Readonly won't work on this.